### ISSUE TITLE
Can't swich language

### ISSUE THREAD

**Issue by duritong** [systemli/systemli-website#23]
I am always redirected to my browser's language.

**Comment by 0x46616c6b** [comment]
Do you still have this issue?

**Comment by duritong** [comment]
Yes, the link DE on /en/ points to / - this redirects me back to /en/

![image](https://user-images.githubusercontent.com/9511/235507574-4875ade5-f133-4091-a1a0-927300475b85.png)

![image](https://user-images.githubusercontent.com/9511/235507813-f0fef731-50e9-41c7-8c55-b3b1a0acb48d.png)
